lib/core/traversalUtils.js

```diff
@@ -51,7 +51,7 @@ export const getRegularClassesMap = (ast: gASTNode): classMapType => {
     fp.map('content'),
     fp.filter({ type: 'ident' }),
     fp.flatMap('content'),
-    fp.filter({ type: 'class' }),
+    fp.filter(node => node.is('class') || node.is('id')),
```

Comment:
Love the initiative! A small suggestion here: This function is intended to return regular classes. What do you think about creating a new one for retrieving `id`s or renaming the function?

Reply:
Renamed the function because it would be a shame to have to go over the ast one more time just for that